packages/react-i18n/src/i18n.ts

```diff
@@ -169,6 +174,22 @@ export default class I18n {
 
     return WEEK_START_DAYS.get(country) || DEFAULT_WEEK_START_DAY;
   }
+
+  @autobind
+  getCurrencySymbol(currencyCode?: string) {
+    const currency = currencyCode || this.defaultCurrency;
+    if (currency == null) {
+      throw new Error(
```

Comment:
Can you create a custom error here: https://github.com/Shopify/quilt/blob/master/packages/react-i18n/src/errors.ts to follow the other errors? 🙏

Reply:
I can re-use `MissingCurrencyCodeError`

Reply:
Awesome 👍